dist/husky.js

```diff
@@ -26702,23 +26702,27 @@ define('type/husky-input',[
             },
 
             typeValidators = {
-                phone:  function(value) {
+                phone: function(value) {
                     var regex = /^[0-9 \\(\\)/+-]*$/;
                     return regex.test(value);
                 },
-                password:  function(value) {
+                password: function(value) {
                     var regex = /^.{6,30}$/;
                     return regex.test(value);
                 },
-                email:  function(value) {
+                email: function(value) {
                     var regex = /^((([a-z]|\d|[!#\$%&'\*\+\-\/=\?\^_`{\|}~]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])+(\.([a-z]|\d|[!#\$%&'\*\+\-\/=\?\^_`{\|}~]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])+)*)|((\x22)((((\x20|\x09)*(\x0d\x0a))?(\x20|\x09)+)?(([\x01-\x08\x0b\x0c\x0e-\x1f\x7f]|\x21|[\x23-\x5b]|[\x5d-\x7e]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])|(\\([\x01-\x09\x0b\x0c\x0d-\x7f]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF]))))*(((\x20|\x09)*(\x0d\x0a))?(\x20|\x09)+)?(\x22)))@((([a-z]|\d|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])|(([a-z]|\d|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])([a-z]|\d|-|\.|_|~|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])*([a-z]|\d|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])))\.)+(([a-z]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])|(([a-z]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])([a-z]|\d|-|\.|_|~|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])*([a-z]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF]))){2,6}$/i;
                     return regex.test(value);
                 },
-                url:  function(value) {
+                url: function(value) {
                     var regex = /^([\da-z\.-]+)\.([a-z\.]{2,6})([\/\w\.-\?\$\#])*\/?/;
                     return regex.test(value);
                 },
-                color:  function(value) {
+                time: function(value) {
+                    var regex = /^([01]?[0-9]|2[0-3]):[0-5][0-9]$/;
```

Comment:
@wachterjohannes this has nothing to do with a german format it's simply a 24 hour time format which should also work internationally. According to Oliver we do not make am/pm formatted time ... If you are asking me why it is in here and not in husky validation then you have to ask the person who implemented this component!